recipe/meta.yaml

```diff
@@ -1,39 +1,33 @@
-{% set version = "1.42.0" %}
+{% set version = "1.62.2" %}
 {% set name = "grpcio-tools" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d0a0daa82eb2c2fb8e12b82a458d1b7c5516fe1135551da92b1a02e2cba93422
-  patches:
-    - 0001-Monkey-patch-distutils.ccompiler.spawn-to-elide-std-.patch
+  sha256: 5fd5e1582b678e6b941ee5f5809340be5e0724691df5299aae8226640f94e18f
 
 build:
-  number: 2
+  skip: true  # [py<37]
+  number: 0
   missing_dso_whitelist:  # [s390x]
     - $RPATH/ld64.so.1    # [s390x]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - patch       # [not win]
-    - m2-patch    # [win]
- 
   host:
-    - grpcio =={{ version }}
     - pip
     - setuptools
     - wheel
-    - protobuf 3.20.3
```

Comment:
That's interesting... `grpcio_tools` "vendors" protobuf (https://github.com/grpc/grpc/blob/v1.62.2/tools/distrib/python/grpcio_tools/setup.py#L193-L200). Fedora patches that so that it uses the system protobuf instead: https://src.fedoraproject.org/rpms/grpc/blob/rawhide/f/grpc.spec#_758.

They also hardcode C++14 (https://github.com/grpc/grpc/blob/v1.62.2/tools/distrib/python/grpcio_tools/setup.py#L140-L151) (though I don't think it matters here).

Reply:
`third_party/abseil-cpp` is also vendored, https://github.com/grpc/grpc/tree/v1.62.2/third_party

Reply:
Added patch to "unvendor" protobuf and abseil. I did not enable it for windows though as I still had issues and do not wish to spend a large amount of time on this (i.e. deferring to later). But, that is still an improvement from before.

Reply:
That's perfect @cbouss. Thanks a lot for giving it a try! Linux + macOS is already a good start.